doc/1chipML_manual.tex

```diff
@@ -576,19 +577,74 @@ \subsection{The Lanczos method}
+\begin{equation}
+    \bar{X} = \frac{1}{N} \; \sum_{i=1}^{N} x_i
+\end{equation}
+The second moment of a distribution is the variance, which measures the dispersion of the data around the mean. It is computed by taking the sum of the squared differences between each data point and the mean, and dividing it by the number of data points minus one. The variance is a very important concept in statistics, as it allows us to quantify the spread of the data. In the library, the formula used to calculate the variance is given by this equation : 
+\begin{equation}
+    \sigma^2 = \frac{1}{N - 1} \; \sum_{i=1}^{N} (x_i - \bar{X})^2
+\end{equation}
+The third moment of a distribution is the skewness, which measures the degree of asymmetry of the data around the mean. A positive skewness indicates that the data has a longer tail on the positive side of the mean, while a negative skewness indicates that the data has a longer tail on the negative side of the mean. In the library, the skewness is implemented using the adjusted Fisher-Pearson standardized moment coefficient, as given by this equation :
+\begin{equation}
+    Skew = \frac{N}{(N - 1)(N - 2)} \; \sum_{i=1}^{N} \Big(\frac{x_i - \bar{X}}{s}\Big)^3
+\end{equation}
+The fourth moment of a distribution is the kurtosis, which measures the degree of peakedness or flatness of the data around the mean. A high kurtosis indicates that the data has more extreme values than a normal distribution, while a low kurtosis indicates that the data has fewer extreme values than a normal distribution. In the library, the kurtosis is implemented using the formula for the excess kurtosis, as given by this equation : 
+\begin{equation}
+    Kurtosis = \frac{N(N + 1)}{(N - 1)(N - 2)(N - 3)} \; \sum_{i=1}^{N} \Big(\frac{x_i - \bar{X}}{s}\Big)^4 \; \frac{3(N - 1)^2}{(N - 2)(N - 3)}
+\end{equation}
+In addition to the four moments, other metrics based on them are also implemented in the library, such as the standard deviation. The standard deviation is the square root of the variance and provides a measure of the average deviation of the data from the mean. It is widely used since it has the same units as the data. In the library, the formula used to calculate the standard deviation is given by this equation : 
+\begin{equation}
+    \sigma = \sqrt{\frac{1}{N - 1} \; \sum_{i=1}^{N} (x_i - \bar{X})^2}
+\end{equation}
+It is important to note that all formulas used to implement the different moments assume that the given data points correspond to a sample of the population and not the population itself. This means that the computed moments are estimates of the true moments of the population, and may be subject to some degree of sampling error.
+
+\subsection{Covariance}
```

Comment:
You've added a `Covariance` section, but did not provide a description. Did you forget to include it?